star_tides/api/controllers/auth_controller.py

```diff
@@ -29,21 +35,19 @@ class CreateUserController(Controller):
         String jwt of current user's login session.
 
     '''
-    def process_request(self):
+    def process_request(self) -> tuple:
```

Comment:
Prefer NamedTuple (subclasses)

Reply:
This is converted into a named tuple because `process_request` is called from the base controller.

Reply:
I think you'll like this way of doing it.